src/pages/event.js

```diff
@@ -431,7 +387,7 @@ const ExternalLinkCell = ({ url, tooltipText = null, content }) => {
   );
 };
 
-function TableContainer({ tokens, ensNames, pageCount: pc, loading }) {
+function TableContainer({ tokens, pageCount: pc, loading }) {
```

Comment:
What's `ensNames` for? Seems that is not used.

Reply:
It's not used that's why I remove it

Reply:
Indeed.